tests/test_integration.py

```diff
@@ -186,3 +186,102 @@ def test_pex_multi_resolve():
     assert len(included_dists) == 4
     for dist_substr in ('-cp27-', '-cp36-', '-manylinux1_x86_64', '-macosx_'):
       assert any(dist_substr in f for f in included_dists)
+
+
+@pytest.mark.xfail
```

Comment:
would be good to pass a `reason=str` arg to xfail for context on why this is expected to fail. linking a ticket etc that describes the issue would be sufficient.

e.g.

```
@pytest.mark.xfail(reason='See http://github.com/...')
```